setup.py

```diff
@@ -28,7 +28,7 @@
     jax_version = [line[4:].strip() for line in f.readlines() if "jax=" in line][0]
 
 requirements = [
-    "pennylane>=0.28",
+    "pennylane @ git+https://github.com/PennyLaneAI/pennylane/@master",
```

Comment:
We need to remember to change to this to the next PennyLane release version

Reply:
Yes.

Reply:
Another for a TODO comment :)

Reply:
https://github.com/PennyLaneAI/catalyst/pull/143/commits/540b379c931696dbd48da6e663790337af264da7